packages/data-portal-explore/src/lib/dataTableHelpers.tsx

```diff
@@ -147,8 +147,7 @@ export function truncatedTableCell<T>(cellData: T) {
     const value = _.isFunction(selector)
         ? selector(cellData)
         : cellData[selector as keyof T];
-
     return value ? (
-        <ExpandableText fullText={value} truncateProps={{ lines: 4 }} />
+        <ExpandableText fullText={value} truncateProps={{ lines: 4 }}/>
```

Comment:
we should just revert the changes in this file for simplicity. we don't really change anything but whitespaces.